.env

```diff
@@ -16,4 +16,5 @@ MINIO_ADDRESS=minio:9000
 PULSAR_ADDRESS=pulsar://pulsar:6650
 ETCD_ENDPOINTS=etcd:2379
 AZURITE_CONNECTION_STRING="DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://azurite:10000/devstoreaccount1;"
+ENABLE_GCP_NATIVE=ON
```

Comment:
sorry I didn't notice here. Please also change to OFF

Reply:
I enable **ENABLE_GCP_NATIVE** to run the GCP-native unit tests (I believe this **.env** file is only used for unit testing purpose). If I turn it **OFF**, how will the GitHub workflow execute the GCP-native-specific unit tests?